src/controllers/albumController.js

```diff
@@ -7,11 +7,35 @@ const createAlbum = async (req, res) => {
   req.body.user = req.user.userId;
   const album = await Album.create(req.body);
   res.status(StatusCodes.CREATED).json({ album });
+  /*
+     #swagger.summary = 'Create new album and save it to the database'
+            #swagger.parameters['newAlbum'] = {
+        in: 'body',
+        description: 'Album information to use for creation.',
+        required: true,
+        type: 'object',
+        schema: { $ref: '#/definitions/NewAlbum' }
+     } 
+     #swagger.responses[201] = {
+				description: 'Albums was successfully created.',
+        schema: [{ $ref: '#/definitions/Album' }]
+		 }
+		 
```

Comment:
This endpoint also has the chance for a `400` error if any validation fails when saving the album (eg: if caller passes in a price that's negative or something)

`#swagger.responses[400] = { description: 'validation error' }`

Reply:
This endpoint can only be used by admins. Let's add a description indicating that:

`#swagger.description = '**ROLE REQUIRED:** admin'`